### ISSUE TITLE
Wrong indices when the domain name contains the same TLD twice

### ISSUE THREAD

**Issue by tkrissuu** [lipoja/URLExtract#109]
I am seeing a strange behaviour when finding URLs with indices. It is triggered when the TLD is also present earlier in the domain name. In the example below ".com" appears twice. But it will also fail with e.g. "www.dk-hostmaster.dk".

To reproduce:
```
from urlextract import URLExtract
extractor = URLExtract()
urls = extractor.find_urls("www.company.com", get_indices=True)
print(urls[0])
```
Output is:
`('www.company.com', (8, 23))`

I would have expected:
`('www.company.com', (0, 15))`

**Comment by lipoja** [comment]
@tkrissuu Thank you for reporting this issue. Nice catch!
I know what is the issue here and I will send fix soon.

**Comment by lipoja** [comment]
@tkrissuu It should be fixed in v1.5.0.
Thanks!

**Comment by tkrissuu** [comment]
@lipoja Awesome. Thanks! That was quick.
